src/Jackett.Common/Definitions/torrentproject.yml

```diff
@@ -0,0 +1,98 @@
+---
+id: torrentproject
+name: TorrentProject
+description: "TorrentProject is a Public torrent meta-search engine"
+language: en-us
+type: public
+encoding: UTF-8
+links:
+  - https://torrentproject.to
+  - https://torrentproject.torrentbay.to/
+
+caps:
+  categorymappings:
+    # site uses › not >
+    - {id: "video › movie", cat: Movies, desc: "Video Movie"}
+    - {id: "video › movie› hdrip", cat: Movies, desc: "Video Movie HDRip"}
+    - {id: "video › movie› dvdrip", cat: Movies/SD, desc: "Video Movie DVDRip"}
+    - {id: audio, cat: Audio, desc: Audio}
+    - {id: "audio › album", cat: Audio, desc: "Audio Album"}
+    - {id: "audio › mp3", cat: Audio/MP3, desc: "Audio MP3"}
+    - {id: "audio › album› mp3", cat: Audio/MP3, desc: "Audio Album MP3"}
+    - {id: "audio › lossless", cat: Audio/Lossless, desc: "Audio Lossless"}
+    - {id: "audio › album› lossless", cat: Audio/Lossless, desc: "Audio Album Lossless"}
+    - {id: software, cat: PC/0day, desc: Software}
+    - {id: mobile, cat: PC/Mobile-Other, desc: Mobile}
+    - {id: games, cat: PC/Games, desc: Games}
+    - {id: "games › pc", cat: PC/Games, desc: "Games PC"}
+    - {id: tv, cat: TV, desc: TV}
+    - {id: video, cat: TV, desc: Video}
+    - {id: "video › show", cat: TV, desc: "Video Show"}
+    - {id: "video › show› movie", cat: TV, desc: "Video Show Movie"}
+    - {id: anime, cat: TV/Anime, desc: Anime}
+    - {id: adult, cat: XXX, desc: Adult}
+    - {id: "video › movie› adult", cat: XXX, desc: "Video Movie Adult"}
+    - {id: "video › show› adult", cat: XXX, desc: "Video Show Adult"}
+    - {id: "› books", cat: Books, desc: Books}
+    - {id: "doc › ebook", cat: Books/EBook, desc: "Doc Ebook"}
+    - {id: other, cat: Other, desc: Other}
```

Comment:
I'm seeing cats in the results like `› games` and `› Software` so I'm thinking it might be better to strip non-alphabetic chars and lowercase it all, to increase the matching.
something like:
```yaml
    category:
      text: other
    category|noappend:
      selector: div > span:nth-child(1) > span:nth-child(2)
      optional: true
      filters:
        - name: re_replace
          args: ["[^a-zA-Z0-9]+", ""]
        - name: tolower
```
then strip the categorymappings
```yaml
  categorymappings:
    - {id: adult, cat: XXX, desc: Adult}
    - {id: anime, cat: TV/Anime, desc: Anime}
    - {id: audio, cat: Audio, desc: Audio}
    - {id: audioalbum, cat: Audio, desc: "Audio Album"}
    - {id: audioalbumlossless, cat: Audio/Lossless, desc: "Audio Album Lossless"}
    - {id: audioalbummp3, cat: Audio/MP3, desc: "Audio Album MP3"}
    - {id: audiolossless, cat: Audio/Lossless, desc: "Audio Lossless"}
    - {id: audiomp3, cat: Audio/MP3, desc: "Audio MP3"}
    - {id: books, cat: Books, desc: Books}
    - {id: docebook, cat: Books/EBook, desc: "Doc Ebook"}
    - {id: games, cat: PC/Games, desc: Games}
    - {id: gamespc, cat: PC/Games, desc: "Games PC"}
    - {id: gamesxbox, cat: Console/XBox, desc: "Games Xbox"}
    - {id: mobile, cat: PC/Mobile-Other, desc: Mobile}
    - {id: other, cat: Other, desc: Other}
    - {id: playstation, cat: Console/PSP, desc: "Games Playstation"}
    - {id: software, cat: PC/0day, desc: Software}
    - {id: tv, cat: TV, desc: TV}
    - {id: video, cat: TV, desc: Video}
    - {id: videomovie, cat: Movies, desc: "Video Movie"}
    - {id: videomovieadult, cat: XXX, desc: "Video Movie Adult"}
    - {id: videomoviedvdrip, cat: Movies/SD, desc: "Video Movie DVDRip"}
    - {id: videomoviehdrip, cat: Movies, desc: "Video Movie HDRip"}
    - {id: videoshow, cat: TV, desc: "Video Show"}
    - {id: videoshowadult, cat: XXX, desc: "Video Show Adult"}
    - {id: videoshowmovie, cat: TV, desc: "Video Show Movie"}
```
I added xbox and playstation too ;-)